src/DssSpell.t.sol

```diff
@@ -1898,29 +1898,81 @@ contract DssSpellTest is DSTest, DSMath {
+        try chainLog.getAddress("MCD_FLIP_LRC_A") returns (address) {
+            assertTrue(false);
+        } catch {}
+
+        assertEq(chainLog.getAddress("MCD_CLIP_BAL_A"), addr.addr("MCD_CLIP_BAL_A"));
+        assertEq(chainLog.getAddress("MCD_CLIP_CALC_BAL_A"), addr.addr("MCD_CLIP_CALC_BAL_A"));
+        try chainLog.getAddress("MCD_FLIP_BAL_A") returns (address) {
             assertTrue(false);
         } catch {}
 
-        assertEq(chainLog.getAddress("MCD_CLIP_ETH_B"), addr.addr("MCD_CLIP_ETH_B"));
-        assertEq(chainLog.getAddress("MCD_CLIP_CALC_ETH_B"), addr.addr("MCD_CLIP_CALC_ETH_B"));
-        try chainLog.getAddress("MCD_FLIP_ETH_B") returns (address) {
+        assertEq(chainLog.getAddress("MCD_CLIP_UNI_A"), addr.addr("MCD_CLIP_UNI_A"));
+        assertEq(chainLog.getAddress("MCD_CLIP_CALC_UNI_A"), addr.addr("MCD_CLIP_CALC_UNI_A"));
+        try chainLog.getAddress("MCD_FLIP_UNI_A") returns (address) {
             assertTrue(false);
         } catch {}
 
-        assertEq(chainLog.getAddress("MCD_CLIP_ETH_C"), addr.addr("MCD_CLIP_ETH_C"));
-        assertEq(chainLog.getAddress("MCD_CLIP_CALC_ETH_C"), addr.addr("MCD_CLIP_CALC_ETH_C"));
-        try chainLog.getAddress("MCD_FLIP_ETH_C") returns (address) {
+        assertEq(chainLog.getAddress("MCD_CLIP_RENBTC_A"), addr.addr("MCD_CLIP_RENBTC_A"));
+        assertEq(chainLog.getAddress("MCD_CLIP_CALC_RENBTC_A"), addr.addr("MCD_CLIP_CALC_RENBTC_A"));
+        try chainLog.getAddress("MCD_FLIP_RENBTC_A") returns (address) {
             assertTrue(false);
         } catch {}
 
-        assertEq(chainLog.getAddress("MCD_CLIP_WBTC_A"), addr.addr("MCD_CLIP_WBTC_A"));
-        assertEq(chainLog.getAddress("MCD_CLIP_CALC_WBTC_A"), addr.addr("MCD_CLIP_CALC_WBTC_A"));
-        try chainLog.getAddress("MCD_FLIP_WBTC_A") returns (address) {
+        assertEq(chainLog.getAddress("MCD_CLIP_AAVE_A"), addr.addr("MCD_CLIP_AAVE_A"));
+        assertEq(chainLog.getAddress("MCD_CLIP_CALC_AAVE_A"), addr.addr("MCD_CLIP_CALC_AAVE_A"));
+        try chainLog.getAddress("MCD_FLIP_AAVE_A") returns (address) {
             assertTrue(false);
         } catch {}
+
+        // This is only for kovan as we are checking keys that were wrong before only in this environment
+        assertEq(chainLog.getAddress("MCD_JOIN_PSM_USDC_A"), addr.addr("MCD_JOIN_PSM_USDC_A"));
+        try chainLog.getAddress("MCD_JOIN_USDC_PSM") returns (address) {
+            assertTrue(false);
+        } catch {}
+        assertEq(chainLog.getAddress("MCD_PSM_USDC_A"), addr.addr("MCD_PSM_USDC_A"));
+        try chainLog.getAddress("MCD_PSM_USDC_PSM") returns (address) {
+            assertTrue(false);
+        } catch {}
+        assertEq(chainLog.getAddress("MCD_FLIP_PSM_USDC_A"), addr.addr("MCD_FLIP_PSM_USDC_A"));
+        try chainLog.getAddress("MCD_FLIP_USDC_PSM") returns (address) {
+            assertTrue(false);
+        } catch {}
+        //
```

Comment:
I think this can be removed, as it was only for kovan

Reply:
Done.